src/modules/popover/popover.md

```diff
@@ -0,0 +1,155 @@
+```jsx
+
+import TextField from '../../elements/forms/text-field';
+import Button from "../../elements/button";
+
+
+  <div
+    style={{
+      margin: '5rem',
+      display: 'grid',
+      gridTemplateColumns: 'repeat(3, 1fr)',
+      gridGap: '0.5rem',
+      gridAutoRows: 'min-content',
+    }}
+  >
+      <Popover placement="top">
+        <Popover.Content>
+          I am a simple tooltip with placement='top'
+        </Popover.Content>
+        <Popover.Trigger
+          tag={Button}
+          variant="outlined"
+          style={{ width: '100%' }}
+        >
+          Top
+        </Popover.Trigger>
```

Comment:
The interface seems to be complicated to me. I like how it's designed but I am not sure you or myself will be happy to use this many times.

This can be used internally but the final interface is desired to be more simple, similarly to `TextField` and `Form.*`. What do you think?

Reply:
Do you have any suggestions?

What about this one?

```jsx
<Tooltip
  content="either text or react lement"
>
<button>Click me!</button>
</Tooltip>
```

Reply:
I like it but not completely. We might want to support rich formatting. In this case, the proposed tooltip will become bloated

```jsx
<Tooltip
  content={(
    <>
      <Icon src="#alert">
      <h3>Error happened<h3>
      <p>Who knows, what is this<p>
    </>
  )}
>
  <button>Click me!</button>
</Tooltip>
```

I propose something slightly different in #290. Let's discuss it there.